contracts/contracts/FleekERC721.sol

```diff
@@ -504,7 +498,7 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl {
     ) public virtual requireTokenRole(tokenId, Roles.Controller) {
         _requireMinted(tokenId);
         _apps[tokenId].builds[++_apps[tokenId].currentBuild] = Build(_commitHash, _gitRepository);
-        emit NewBuild(tokenId, _commitHash, msg.sender);
+        emit MetadataUpdate(tokenId, "build", _commitHash, msg.sender);
```

Comment:
But changing repository should be handled differently right? The way I imagine is you have different events and the handler updates different fields on the same entity, like
```
MetadataUpdate(1, "build", "3a4c5b8...", me)    ->     handles commit hash change -> update entity's commit hash
MetadataUpdate(1, "repo", "fleekxyz/nfa", me)  ->     handles repo change               -> update entity's repo
```
Because new commit hash and new repo won't happen at the same time.

Reply:
> But changing repository should be handled differently right? The way I imagine is you have different events and the handler updates different fields on the same entity, like
> 
> ```
> MetadataUpdate(1, "build", "3a4c5b8...", me)    ->     handles commit hash change -> update entity's commit hash
> MetadataUpdate(1, "repo", "fleekxyz/nfa", me)  ->     handles repo change               -> update entity's repo
> ```
> 
> Because new commit hash and new repo won't happen at the same time.

Right now is a single update, when we set the a build it requires both parameters https://github.com/fleekxyz/non-fungible-apps/blob/main/contracts/contracts/FleekERC721.sol#L500

Reply:
IMO this makes sense since we are tracking always repo and commit to new builds in the history.

Reply:
We don't need to do anything in this PR but my note was more around being able to support repo changes without it being a new build necessarily.  Right now we don't support changing the repo alone but when we add it, the handler should be updating the graph with just the repo change.

Reply:
My comment was meant to be taken for the current implementation. If we let github repo change and that change to happen separately from a new commit hash, that's something else.